<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QMainWindow" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eos.toolkit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>198</red>
         <green>194</green>
         <blue>189</blue>
        </color>
       </brush>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Debugger">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="actionMessages_Log_Inspector"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionScript_Editor"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Debugger"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Application</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Resources</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="tabSpacing">
            <number>3</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>256</width>
              <height>248</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Scripts</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidget"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>96</width>
              <height>26</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionMessages_Log_Inspector">
   <property name="text">
    <string>Messages Log Inspector</string>
   </property>
  </action>
  <action name="actionScript_Editor">
   <property name="text">
    <string>Script Editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
